packages/happner-2/test/integration/security/user-group-management-mongo.js

```diff
@@ -373,7 +373,7 @@ require('../../__fixtures/utils/test_helper').describe({ timeout: 120e3 }, (test
     });
   });
 
-  it('adds a test user, logs in with the test user - modifies the user details without the users password changing', function (done) {
+  it.skip('adds a test user, logs in with the test user - modifies the user details without the users password changing', function (done) {
```

Comment:
maybe just delete these instead of skipping them?

Reply:
Removed